.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ permissions:
   contents: read
 
 env:
-  RUSTFLAGS: -D warnings -F unused_must_use
+  RUSTFLAGS: -D warnings
```

Comment:
Slightly unclear whether this belongs in this commit?

Reply:
It was causing issues with the `allow(unused)` I added IIRC. Don't think we need the `-F` anyway when we have `-D warnings`?

Reply:
True!